From 998b4240bf1af0327df5ef9ff88376f4ef3b3c55 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Thu, 26 Mar 2020 11:23:14 +0800
Subject: [PATCH] ARM: dts: rv1126: Fix wrong sdmmc and sdio pinctrl

pinctrl was reordered in rv1126-pinctrl.dtsi and swap the
ID of sdmmc0_* with sdmmc1_* but didn't update rv1126.dtsi.

Change-Id: I73373c915e329b387450d5c96869a8d8ee9a95ec
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm/boot/dts/rv1126.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index 50a52399deb2..63e5397cc983 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -1451,7 +1451,7 @@
 		fifo-depth = <0x100>;
 		max-frequency = <200000000>;
 		pinctrl-names = "default";
-		pinctrl-0 = <&sdmmc1_clk &sdmmc1_cmd &sdmmc1_det &sdmmc1_bus4>;
+		pinctrl-0 = <&sdmmc0_clk &sdmmc0_cmd &sdmmc0_det &sdmmc0_bus4>;
 		status = "disabled";
 	};
 
@@ -1465,7 +1465,7 @@
 		fifo-depth = <0x100>;
 		max-frequency = <200000000>;
 		pinctrl-names = "default";
-		pinctrl-0 = <&sdmmc0_clk &sdmmc0_cmd &sdmmc0_bus4>;
+		pinctrl-0 = <&sdmmc1_clk &sdmmc1_cmd &sdmmc1_bus4>;
 		power-domains = <&power RV1126_PD_SDIO>;
 		status = "disabled";
 	};
-- 
2.35.3

